lib/tasks/govuk_lint.rake

```diff
@@ -1,10 +1,11 @@
-desc "Lint ruby code"
 namespace :lint do
+  desc "Lint ruby code"
```

Comment:
these descriptions, tasks and also the rspec strings should start with lower case letters. I think we agreed this a while ago but probably didn't get a linting rule in for it.

Reply:
@gpeng are we sure for the rake tasks?

![image](https://user-images.githubusercontent.com/470137/91966939-74028700-ed0a-11ea-8703-d9bb2aa5cba5.png)


all starts with upper case a part from lint...

Reply:
🤔 I'm easy either way as long as we're consistent in the ones we add.